training-node/app/controllers/user.js

```diff
@@ -0,0 +1,55 @@
+const userServices = require('../services/user'),
```

Comment:
When you branch from something that has not been merged, you should compare to that branch and not to master in a Pull Request in order to see only the changes you made. You can change the base branch easily in the pr edition.

Reply:
Done !